tests/Oryx.BuildImage.Tests/BenvTest.cs

```diff
@@ -125,41 +125,7 @@ public void UserInstalledExecutable_IsChosenOverOryxExecutable(string executable
                 .ToString();
 
             // Act
-            var image = _imageHelper.GetBuildImage();
-            var result = _dockerCli.Run(image, "/bin/bash", "-c", script);
-
-            // Assert
-            RunAsserts(
-                () =>
-                {
-                    Assert.True(result.IsSuccess);
-                    Assert.Contains(userInstalledExecutable, result.StdOut);
-                },
-                result.GetDebugInfo());
-        }
-
-        [Theory]
-        [InlineData("dotnet")]
-        [InlineData("node")]
-        [InlineData("npm")]
-        [InlineData("npx")]
-        [InlineData("yarn")]
-        [InlineData("python")]
-        [InlineData("php")]
-        public void UserInstalledExecutable_IsChosenOverOryxExecutable_InLtsVersionsBuildImage(string executableName)
```

Comment:
we need a new test to replace this instead of deleting this?

Reply:
Do we want to still keep this test?

Reply:
I deleted this test because it does not apply anymore since by default Oryx versions are supposed to be picked up in case of ltsversions, full build image.